examples/demo_dlmbl/solution.py

```diff
@@ -389,74 +467,67 @@ def log_batch_jupyter(batch):
 # %%
 log_batch_jupyter(augmented_batch)
 
-# %% [markdown]
-# <div class="alert alert-info">
-#
-# ### Task 1.3
-# Can you tell what augmentation were applied from looking at the augmented images in Tensorboard?
-#
-# Check your answer using the source code [here](https://github.com/mehta-lab/VisCy/blob/b89f778b34735553cf155904eef134c756708ff2/viscy/light/data.py#L529).
-# </div>
-
 # %% [markdown]
 """
 ## Train a 2D U-Net model to predict nuclei and membrane from phase.
 
-### Construct a 2D U-Net
+### Construct a 2D UNeXt2 using VisCy
 See ``viscy.unet.networks.Unet2D.Unet2d`` ([source code](https://github.com/mehta-lab/VisCy/blob/7c5e4c1d68e70163cf514d22c475da8ea7dc3a88/viscy/unet/networks/Unet2D.py#L7)) for configuration details.
 """
 # %%
 # Create a 2D UNet.
-GPU_ID = 0
-BATCH_SIZE = 10
-YX_PATCH_SIZE = (512, 512)
+GPU_ID = 1
+BATCH_SIZE = 12
+YX_PATCH_SIZE = (256, 256)
 
 
 # Dictionary that specifies key parameters of the model.
-phase2fluor_config = {
-    "architecture": "2D",
-    "num_filters": [24, 48, 96, 192, 384],
-    "in_channels": 1,
-    "out_channels": 2,
-    "residual": True,
-    "dropout": 0.1,  # dropout randomly turns off weights to avoid overfitting of the model to data.
-    "task": "reg",  # reg = regression task.
-}
+phase2fluor_config=dict(
+    in_channels=1,
+    out_channels=2,
+    encoder_blocks=[3, 3, 9, 3],
+    dims=[96, 192, 384, 768],
+    decoder_conv_blocks=2,
+    stem_kernel_size=(1, 2, 2),
+    in_stack_depth=1,
+    pretraining=False,
+)
 
 phase2fluor_model = VSUNet(
+    architecture='fcmae', #2D UNeXt2 architecture
```

Comment:
UNeXt2 architecture and VSUNet module should be fixed so that this reads architecture = `unext2`